.github/actions/pytest_run/action.yml

```diff
@@ -6,7 +6,8 @@ runs:
     - name: Test with pytest
       run: |
         python -m pytest -rx -m "not parallel and c" --ignore=ast --ignore=printers --ignore=symbolic --ignore=ndarrays
-        python -m pytest -rx -m "not parallel and not c" --ignore=ast --ignore=printers --ignore=symbolic --ignore=ndarrays
+        python -m pytest -rx -m "not parallel and not c and not python" --ignore=ast --ignore=printers --ignore=symbolic --ignore=ndarrays
```

Comment:
Could we just pass the flag `-m "not parallel and fortran"`? I think we use the `fortran` mark quite consistently now.

Reply:
Unfortunately not. The `fortran` mark is currently being added thanks to the `language` fixture. However no `fortran` marks have been added manually to functions that only run in one language (e.g. `test_arrays.py` currently).

If we want to do this at some stage it might also be useful to add another mark for language independent tests such as the error tests, `test_arrays_multiple_assignments.py`, etc

Reply:
I see, you're right. We may need to rethink how we manage multiple languages in our unit tests.